src/Styling/Book.css

```diff
@@ -0,0 +1,15 @@
+.book-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.book-info-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.book-actions-container {
+  display: flex;
```

Comment:
- [ ] For DRY CSS kindly combine duplicate properties. 
Example, you can create class `.d-flex { display: flex}` and use this class in HTML file.
_This is just one example. Please check your CSS file and if some CSS rules are the same please combine them also._
- [More information about DRY principles](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) 

- [Html and CSS best practices in curriculum](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=If%20you%20want%20to%20create%20a%20CSS%20rule%20for%20multiple%20elements%2C%20use%20a%20class%20attribute%20instead.%20For%20example%2C%20instead%20of%20using%20color%3A%20%233498db%3B%203%20times)